crt_portal/cts_forms/tests/integration/report_submission.py

```diff
@@ -0,0 +1,142 @@
+
+import pytest
+
+
+@pytest.mark.only_browser("chromium")
+def test_report_complete_and_valid_submission(page):
+    page.goto("/report")
+    assert page.title() == "Step 1: Contact - Contact the Civil Rights Division | Department of Justice"  # nosec
+
+    # Click input[name="0-contact_first_name"]
+    page.click("input[name=\"0-contact_first_name\"]")
```

Comment:
Wouldn't it be less error prone to use `'`s here to avoid escaping double quotes?